src/CTA.Rules.Models/Enums.cs

```diff
@@ -3,6 +3,7 @@
     public enum ActionTypes
     {
         Method,
+        Statement,
```

Comment:
Can this be expression? Just to stay consistent

Reply:
Sure thing, no problem.